polaris.shopify.com/content/design/icons/using-icons.md

```diff
@@ -0,0 +1,169 @@
+
+</Grid>
+
+<Card>
+   
+  <Grid gap='4'>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      ### Replacing standard actions
+
+      Common actions like edit, delete, and search have established icons that are universally understood. Leveraging pre-existing knowledge will improve clarity and help reduce clutter, since these icons require no text label.
+
+    </Grid.Cell>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      <Do>
+        ![](/images/design/icons/icons-using-2@2x.png)
+
+        Use icons to replace text for common actions with universally understood icons.
+
+      </Do>
+
+    </Grid.Cell>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      <Dont>
+        ![](/images/design/icons/icons-using-2@2x.png)
+
+        Use ambiguous icons without a text label, as that can lead to confusion and misinterpretation.
+
+      </Dont>
+
+    </Grid.Cell>
+
+  </Grid>
+
+</Card>
+
+<Card>
+   
+  <Grid gap='4'>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      ### Indicate common objects
+
+      Consistently pairing specific icons with objects enables users to quickly recognize them, which promotes a more efficient and intuitive interaction with the interface.
+
+    </Grid.Cell>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 8, xl: 8}}>
+
+      <Grid gap='4'>
+        <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 6, xl: 6}}>
+           <Do>
+              ![](/images/design/icons/icons-using-4@2x.png)
+
+              Use icons consistently once a meaning is assigned to them.
+
+            </Do>
+        </Grid.Cell>
+        <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 6, xl: 6}}>
+          <Do>
+            ![](/images/design/icons/icons-using-5@2x.png)
+
+            Use icons consistently once a meaning is assigned to them.
+
+          </Do>
+        </Grid.Cell>
+
+        <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 12, xl: 12}}>
+          <Dont>
+            ![](/images/design/icons/icons-using-6@2x.png)
+
+            Use icons consistently once a meaning is assigned to them.
+
+          </Dont>
+        </Grid.Cell>
+
+      </Grid>
+
+    </Grid.Cell>
+
+  </Grid>
+
+</Card>
+
+<Card>
+   
+  <Grid gap='4'>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      ### Size
+
+      Icon size and weight was specifically crafted to pair with typography.
+
+    </Grid.Cell>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      <Do>
+        ![](/images/design/icons/icons-using-7@2x.png)
+
+        Use icons to replace text for common actions with universally understood icons.
+
+      </Do>
+
+    </Grid.Cell>
+
+    <Grid.Cell columnSpan={{xs: 6, sm: 6, md: 6, lg: 4, xl: 4}}>
+
+      <Dont>
+        ![](/images/design/icons/icons-using-8@2x.png)
+
+        Use ambiguous icons without a text label, as that can lead to confusion and misinterpretation.
```

Comment:
Replace this string with:
Resize icons. Resizing them will distort the relationship that was established with typography.